src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx

```diff
@@ -103,13 +104,18 @@ export const LanguageEngagementListItemCard: FC<LanguageEngagementListItemCardPr
             <Grid item>
               <Typography variant="h4">{name}</Typography>
             </Grid>
-            {registryOfDialectsCode ? (
-              <Grid item>
-                <Typography variant="body2" color="textSecondary">
-                  Registry Of Dialects Code: {registryOfDialectsCode}
-                </Typography>
-              </Grid>
-            ) : null}
+            <Grid item>
+              <DisplaySimpleProperty
+                label="Registry Of Dialects Code"
+                value={registryOfDialectsCode}
+              />
+            </Grid>
```

Comment:
Can you change these to use the new `wrap` prop? This way the grid item doesn't get rendered if the value doesn't exist.
```suggestion
            <DisplaySimpleProperty
              label="Registry Of Dialects Code"
              value={registryOfDialectsCode}
              wrap={(node) => <Grid item>{node}</Grid>}
            />
```

Reply:
Done, had to change up the `DisplaySimpleProperty` wrap rending though because it was still calling the wrap if the value is empty